csharp/ql/test/library-tests/generics/Generics.expected

```diff
@@ -174,5 +178,142 @@ test30
+| Nesting.cs:15:21:15:27 | MC2<> | A<System.String>+C.MC2<> | A<System.String>+C.MC2<T5>(string, T5) |
+| Nesting.cs:17:22:17:26 | D<> | A<>+C+D<> | A<>+C+D<T6> |
+| Nesting.cs:17:22:17:26 | D<> | A<System.Int32>+C+D<> | A<System.Int32>+C+D<T6> |
+| Nesting.cs:17:22:17:26 | D<> | A<System.String>+C+D<> | A<System.String>+C+D<T6> |
+| Nesting.cs:20:25:20:31 | MD2<> | A<>+C+D<>.MD2<> | A<>+C+D<>.MD2<T7>(T1, T6, T7) |
+| Nesting.cs:20:25:20:31 | MD2<> | A<System.Int32>+C+D<System.Boolean>.MD2<> | A<System.Int32>+C+D<System.Boolean>.MD2<T7>(int, bool, T7) |
+| Nesting.cs:20:25:20:31 | MD2<> | A<System.String>+C+D<System.Decimal>.MD2<> | A<System.String>+C+D<System.Decimal>.MD2<T7>(string, decimal, T7) |
+| generics.cs:16:27:16:58 | GenericDelegateInGenericClass<> | generics.A<>+GenericDelegateInGenericClass<> | generics.A<>+GenericDelegateInGenericClass<U>(T, U) |
+| generics.cs:16:27:16:58 | GenericDelegateInGenericClass<> | generics.A<System.Int32>+GenericDelegateInGenericClass<> | generics.A<System.Int32>+GenericDelegateInGenericClass<U>(int, U) |
+| generics.cs:16:27:16:58 | GenericDelegateInGenericClass<> | generics.A<System.String>+GenericDelegateInGenericClass<> | generics.A<System.String>+GenericDelegateInGenericClass<U>(string, U) |
+| generics.cs:18:18:18:23 | bar<> | generics.A<>.bar<> | generics.A<>.bar<X>(X, T) |
+| generics.cs:18:18:18:23 | bar<> | generics.A<System.Int32>.bar<> | generics.A<System.Int32>.bar<X>(X, int) |
+| generics.cs:18:18:18:23 | bar<> | generics.A<System.String>.bar<> | generics.A<System.String>.bar<X>(X, string) |
+| generics.cs:45:14:45:17 | f<> | generics.B<>.f<> | generics.B<>.f<X>() |
+| generics.cs:45:14:45:17 | f<> | generics.B<System.Object>.f<> | generics.B<System.Object>.f<X>() |
+| generics.cs:45:14:45:17 | f<> | generics.B<System.String>.f<> | generics.B<System.String>.f<X>() |
+| generics.cs:45:14:45:17 | f<> | generics.B<X>.f<> | generics.B<X>.f<X>() |
+| generics.cs:51:22:51:34 | Inner<,> | generics.Outer<,>+Inner<,> | generics.Outer<,>+Inner<U1, U2> |
+| generics.cs:51:22:51:34 | Inner<,> | generics.Outer<System.Object,System.Boolean>+Inner<,> | generics.Outer<System.Object,System.Boolean>+Inner<U1, U2> |
+| generics.cs:51:22:51:34 | Inner<,> | generics.Outer<System.Object,System.Int32>+Inner<,> | generics.Outer<System.Object,System.Int32>+Inner<U1, U2> |
+| generics.cs:56:25:56:40 | MyMethod<,> | generics.Outer<,>+Inner<,>.MyMethod<,> | generics.Outer<,>+Inner<,>.MyMethod<W1, W2>(W1, W2, U2, T2) |
+| generics.cs:56:25:56:40 | MyMethod<,> | generics.Outer<System.Object,System.Boolean>+Inner<System.Object,System.String>.MyMethod<,> | generics.Outer<System.Object,System.Boolean>+Inner<System.Object,System.String>.MyMethod<W1, W2>(W1, W2, string, bool) |
+| generics.cs:56:25:56:40 | MyMethod<,> | generics.Outer<System.Object,System.Int32>+Inner<System.String,System.Int32>.MyMethod<,> | generics.Outer<System.Object,System.Int32>+Inner<System.String,System.Int32>.MyMethod<W1, W2>(W1, W2, int, int) |
```

Comment:
The same inconsistency is now present with methods:
`<System.String,System.Int32>` vs `<W1, W2>`

Also, there's a difference between types:
`System.Int32` vs `int`